README.md

```diff
@@ -189,17 +211,47 @@ EXAMPLES
   ... 'devenv-github-workflows' is up to date!
   ... 'devenv-github-templates' is up to date!
 ```
-
-_See code: [dist/commands/upgrade/index.ts](./src/commands/update/index.ts)_
```

Comment:
Same here, why no reference?
We could also think of just removing the references completely